schulcloud/CalendarEventHelper+EventKit.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  CalendarEventHelper+EventKit.swift
+//  schulcloud
+//
+//  Created by Florian Morel on 19.01.18.
+//  Copyright © 2018 Hasso-Plattner-Institut. All rights reserved.
+//
+
+import Foundation
+import EventKit
+import BrightFutures
+
+
+// MARK: Extension with EvenKit convenience
+extension CalendarEventHelper {
+    
+    private static var eventStore: EKEventStore = EKEventStore()
+    private static var calendar : EKCalendar?
+
+    private struct Keys {
+        static let isSynchonized = "org.schul-cloud.calendar.eventKitIsSynchonized"
+        static let calendarIdentifier = "org.schul-cloud.calendar.identifier"
+    }
+    
+    struct EventKitSettings {
+        
+        static var current : EventKitSettings = EventKitSettings()
+        var isSynchonized : Bool {
```

Comment:
I would prefer `shouldSynchronize`. Please also change to name of the key.

Reply:
That's more descriptive of what is going indeed. Will update.